t/prometheus-metrics.t

```diff
@@ -424,3 +424,70 @@ qr/upstream_status\{status="200",service_id="21",service_system_name="bar"\} 1/
 ]]
 --- no_error_log
 [error]
+
+
+=== TEST 8: APICast status metric
+This metric report APICast status, because service cannot be found, auth is
```

Comment:
This metric reports APIcast status in the following cases: when the service cannot be found, auth is